src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/repository/ReferralRepository.kt

```diff
@@ -20,4 +24,73 @@ interface ReferralRepository : JpaRepository<Referral, UUID> {
   // queries for draft referrals
   fun findByIdAndSentAtIsNull(id: UUID): Referral?
   fun findByCreatedByIdAndSentAtIsNull(userId: String): List<Referral>
+
+  @Query(value =
+    "WITH attended_sessions AS ( " +
```

Comment:
you can use triple quotes for multiline strings to avoid all the extra quotes and plus symbols.